bluesky/run_engine.py

```diff
@@ -42,6 +44,16 @@ class _RunEnginePanic(Exception):
     ...
 
 
+@dataclass
+class RunEngineResult():
+    run_start_uids: typing.Tuple[str]
```

Comment:
```suggestion
    run_start_uids: typing.Tuple[str, ...]
```

This change allows variable length tuples by default the existing code would have only type checked for 1-tuples

Reply:
Oh great, thanks for catching. I'm a bit new at type hints, and I guess they aren't actually checked by anything in the tests.